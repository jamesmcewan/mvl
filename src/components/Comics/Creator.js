const Creator = (creator) => (
  <li class="list-none text-yellow-200" key={`${creator.name}-${creator.role}`}>
    {creator.name} - {creator.role}
  </li>
)

export default Creator
