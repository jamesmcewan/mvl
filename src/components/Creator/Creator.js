import React from 'react';

const Creator = ({ name, role }) => (
  <li>
    {name} - {role}
  </li>
);

export default Creator;
