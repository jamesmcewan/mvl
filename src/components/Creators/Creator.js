const Creator = (creator) => (
  <li
    className="list-none text-yellow-200"
    key={`${creator.name}-${creator.role}`}
  >
    {creator.name} - {creator.role}
  </li>
)

export default Creator
