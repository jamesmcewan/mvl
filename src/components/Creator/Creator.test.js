import React from 'react';
import { mount } from 'enzyme';
import Creator from './Creator';
import 'jest-styled-components';

describe('Creator', () => {
  it('should render without crashing', () => {
    const wrapper = mount(<Creator />);
    expect(wrapper).toHaveLength(1);
  });

  it('should render the correct JSX', () => {
    const wrapper = mount(<Creator />);
    expect(wrapper).toMatchSnapshot();
  });
});
