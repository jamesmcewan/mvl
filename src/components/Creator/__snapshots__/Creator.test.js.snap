// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creator should render the correct JSX 1`] = `
<Creator>
  <li>
     - 
  </li>
</Creator>
`;
